models/item.js

```diff
@@ -0,0 +1,37 @@
+const { required } = require('joi')
+const mongoose = require('mongoose')
+
+const ItemShcema = new mongoose.Schema({
+    itemNumber: {
+        type: String,
+        required: [true, "Please provide item number"]
+    },
+    itemName: {
+        type: String,
+        required: [true, 'Please provide item name'],
+        maxLength: 50,
+    },
+    price: {
+        type: Number,
```

Comment:
Consider adding in some additional validations here.  For example, price should always be positive

Reply:
I added min: 1 , validator. I tested it and it resolves the positive validator issue.